openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -786,12 +786,19 @@ const TagsPage = () => {
         ) : (
           <div className="full-height" data-testid="tags-container">
             {currentClassification && (
-              <Space
-                align="start"
-                className="w-full justify-between"
-                data-testid="header">
-                <Space align="end">
+              <Row data-testid="header" wrap={false}>
+                <Col flex="auto">
                   <EntityHeaderTitle
+                    badge={
+                      currentClassification.provider ===
+                        ProviderType.System && (
+                        <AppBadge
+                          className="m--t-xss"
+                          icon={<LockIcon height={12} />}
+                          label={capitalize(currentClassification.provider)}
+                        />
+                      )
```

Comment:
```suggestion
                      currentClassification.provider ===
                        ProviderType.System ? (
                        <AppBadge
                          className="m--t-xss"
                          icon={<LockIcon height={12} />}
                          label={capitalize(currentClassification.provider)}
                        /> : null
                      )
```

To avoid extra things to be render like `false` or `true`